### PR TITLE
Missions - updated text for Fermi, Swift, LVK

### PR DESCRIPTION
**Pull request by eburnsastro**
Minor changes, additional table each for Fermi, Swift LVK. Commits from June 2nd on should be the only ones that matter (I pulled from the main fork/repo yesterday).

### CODE REVIEW

File: app/routes/missions/swift.md [hunk 1]
```diff
@@ -25,39 +25,52 @@ https://swift.gsfc.nasa.gov/archive/
...
+**Swift GRB Annual Trigger Detection Rates:**
+
+| Instrument   | Long GRBs | Short GRBs | Description                   |
+| ------------ | --------- | ---------- | ----------------------------- |
+| `SWIFT_BAT`  | 70-80     | 7-9        | On-board trigger rate         |
+| `SWIFT_XRT`  | 60-65     | 40-50      | XRT follow-up detections      |
+| `SWIFT_UVOT` | 20-25     | 0-1        | UVOT follow-up detections     |
+| `SWIFT_BAT`  | 8-10      | 2-4        | GUANO arcminute recovery rate |
+
+Trigger Rates determined from catalogued information including the [Swift-BAT Third GRB Catalog](https://swift.gsfc.nasa.gov/results/batgrbcat/index.html) and the [Swift GRB Lookup Table](https://swift.gsfc.nasa.gov/archive/grb_table/). GUANO results are from the [online summary table](https://www.swift.psu.edu/guano/).
```
Comment (lpsinger) [inline app/routes/missions/swift.md:76]: Switch to the more common American English spelling:

```suggestion
Trigger Rates determined from cataloged information including the [Swift-BAT Third GRB Catalog](https://swift.gsfc.nasa.gov/results/batgrbcat/index.html) and the [Swift GRB Lookup Table](https://swift.gsfc.nasa.gov/archive/grb_table/). GUANO results are from the [online summary table](https://www.swift.psu.edu/guano/).
```

File: app/routes/missions/swift.md [hunk 2]
```diff
@@ -25,39 +25,52 @@ https://swift.gsfc.nasa.gov/archive/
...
+| `SWIFT_BAT_KNOWN_SRC`     | 1–12 h                      | 1–7′               | Peaks of known-sources found in all image produced on-board |
+
+**Swift GRB Annual Trigger Detection Rates:**
+
+| Instrument   | Long GRBs | Short GRBs | Description                   |
+| ------------ | --------- | ---------- | ----------------------------- |
+| `SWIFT_BAT`  | 70-80     | 7-9        | On-board trigger rate         |
+| `SWIFT_XRT`  | 60-65     | 40-50      | XRT follow-up detections      |
+| `SWIFT_UVOT` | 20-25     | 0-1        | UVOT follow-up detections     |
+| `SWIFT_BAT`  | 8-10      | 2-4        | GUANO arcminute recovery rate |
```
Comment (lpsinger) [inline app/routes/missions/swift.md:74]: No need to typeset instrument names in `monospace` as if they were code.

Also, use en dashes (–, option+dash on macOS) for numerical ranges.

```suggestion
| BAT | 70–80     | 7–9        | On-board trigger rate         |
| XRT  | 60–65     | 40–50      | XRT follow-up detections      |
| UVOT | 20–25     | 0–1        | UVOT follow-up detections     |
| `SWIFT_BAT`  | 8–10      | 2–4        | GUANO arcminute recovery rate |
```

File: app/routes/missions/fermi.md [hunk 3]
```diff
@@ -43,3 +44,16 @@ https://fermi.gsfc.nasa.gov/ssc/data/
...
+**Fermi Yearly Trigger Rates:**
+
+| Instrument  | Type                             | Rates |
+| ----------- | -------------------------------- | ----- |
+| `FERMI_GBM` | `Short GRB`                      | 35-40 |
+|             | `Long GRB`                       | 200   |
+|             | `Soft Gamma-ray Repeater Flares` | 35-40 |
+|             | `Terrestrial Gamma-ray Flashes`  | 80-90 |
+|             | `Solar Flares`                   | 90-95 |
+| `FERMI_LAT` | `GRB`                            | 15-20 |
```
Comment (lpsinger) [inline app/routes/missions/fermi.md:57]:
```suggestion
| Instrument  | Type                             | Rates |
| ----------- | -------------------------------- | ----- |
| GBM | Short GRB                      | 35–40 |
|             | Long GRB                       | 200   |
|             | Soft Gamma-ray Repeater Flares | 35–40 |
|             | Terrestrial Gamma-ray Flashes  | 80–90 |
|             | Solar Flares                   | 90–95 |
| LAT | GRB                            | 15–20 |
```

**Review by lpsinger — CHANGES_REQUESTED** [review]

File: app/routes/missions/lvk.md [hunk 4]
```diff
@@ -53,3 +51,13 @@ O4 predictions from [Abbott et al. 2020, LRR](https://link.springer.com/article/
...
 | `LVC_UPDATE`      | Ultimate refined skymap (more detailed off-line processing).                                  | ~1–7 d   |
 | `LVC_RETRACTION`  | After human analysis/evaluation, a retraction will be issued if trigger is not astrophysical. | 1 h–1 d  |
+
+**Predicted Detection Rates for O4:**
+
+| Merger Class              | Detection Rates per Calendar Year | Area [deg<sup>2</sup>] |
+| ------------------------- | --------------------------------- | ---------------------- |
+| `Binary Neutron Star`     | 10 (0–62)                         | 33 (28–38)             |
+| `Neutron Star-Black Hole` | 1 (0–92)                          | 50 (42–58)             |
+| `Binary Black Hole`       | 79 (35–168)                       | 41 (35–48)             |
```
Comment (lpsinger) [inline app/routes/missions/lvk.md:61]: No need to set in teletype/monospace font here:

```suggestion
| Binary Neutron Star       | 10 (0–62)                         | 33 (28–38)             |
| Neutron Star-Black Hole   | 1 (0–92)                          | 50 (42–58)             |
| Binary Black Hole         | 79 (35–168)                       | 41 (35–48)             |
```

**Review by lpsinger — CHANGES_REQUESTED** [review]

**Comment by lpsinger** [comment]
@eburnsastro, I've approved this, but GitHub won't let me merge it because there are conflicts. Would you please resolve the merge conflicts?

**Comment by eburnsastro** [comment]
> 

@lpsinger I do not see merge conflicts. Nothing appears on this page and pulling from the tachgsfc/gcn.nasa.gov main did not identify any. I pulled this and pushed back here but still see nothing. If you let me know how to see the merge conflict (Google only brought the previous options) I can work on fixing it

**Comment by lpsinger** [comment]
Hmm, it still says that there are merge conflicts.

**Comment by eburnsastro** [comment]
I see this when attempting to find merge conflicts:
(base) eburns@heimdall:~/tach/my_fork/gcn.nasa.gov$ git checkout missions
Switched to branch 'missions'
(base) eburns@heimdall:~/tach/my_fork/gcn.nasa.gov$ git fetch upstream
(base) eburns@heimdall:~/tach/my_fork/gcn.nasa.gov$ git checkout main
Switched to branch 'main'
Your branch is up to date with 'origin/main'.
(base) eburns@heimdall:~/tach/my_fork/gcn.nasa.gov$ git merge upstream/main
Already up to date.
(base) eburns@heimdall:~/tach/my_fork/gcn.nasa.gov$ git push origin missions
Everything up-to-date

Suggestions?

**Comment by lpsinger** [comment]
I rebased it by hand and merged it. Thanks!

**Pull request merged**